e2e/pages/k8s-page.ts

```diff
@@ -0,0 +1,18 @@
+import { Page, Locator } from 'playwright';
+import { expect } from '@playwright/test';
+
+export class K8sPage {
+    readonly page: Page;
+    readonly k8sMemorySliderSelector: Locator;
```

Comment:
Why are these things called `…Selector`? Compared to, say, `k8sResetBtn` (as opposed to `k8sResetBtnSelector`).

Do we even need the `k8s` prefix, given that it's hanging off the K8s page?

Reply:
Thanks for calling this out. I'll improve it a bit more.